pyproject.toml

```
@@ -24,23 +24,15 @@ Changelog = "https://github.com/statisticsnorway/datadoc/releases"
 
 [tool.poetry.dependencies]
 python = ">=3.10,<4.0"
-pyarrow = ">=8.0.0"
 dash = ">=2.15.0"
 pydantic = ">=2.5.2"
 dash-bootstrap-components = ">=1.1.0"
-pandas = ">=1.4.2"
-dapla-toolbelt = ">=1.3.3"
-gunicorn = ">=21.2.0"
 flask-healthz = ">=0.0.3"
 arrow = ">=1.3.0"
 python-dotenv = ">=1.0.1"
-requests = ">=2.31.0"
-beautifulsoup4 = ">=4.12.3"
-cloudpathlib = { extras = ["gs"], version = ">=0.17.0" }
-pyjwt = ">=2.8.0"
-ssb-klass-python = ">=0.0.9"
 ssb-dash-components = ">=0.8.1"
-ssb-datadoc-model = "^6.0.0"
+dapla-toolbelt-metadata = "^0.2.0"
+gunicorn = "^23.0.0"
```

```suggestion
dapla-toolbelt-metadata = ">=0.2.0"
gunicorn = ">=23.0.0"
```

We should use the `>=` constraint like the other dependencies so we're not restricting upgrades unnecessarily.